project/play-fork-run.sbt

```diff
@@ -0,0 +1,3 @@
+// This plugin adds forked run capabilities to Play projects which is needed for Activator.
+
+addSbtPlugin("com.typesafe.play" % "sbt-fork-run-plugin" % "2.3.8")
```

Comment:
I cloned the repo, but when running via sbt I got
```
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	::          UNRESOLVED DEPENDENCIES         ::
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
[warn] 	:: com.typesafe.play#sbt-fork-run-protocol_2.10;2.3.8: not found
[warn] 	:: com.typesafe.play#routes-compiler_2.10;2.3.8: not found
[warn] 	:: com.typesafe.play#sbt-run-support_2.10;2.3.8: not found
[warn] 	::::::::::::::::::::::::::::::::::::::::::::::
``` 
in order to fix it I've updated the dependency 
`addSbtPlugin("com.typesafe.play" % "sbt-fork-run-plugin" % "2.4.4")`
I hope it helps